ONSdigital/eq-author-app
# Bug - Laggy routing number input

### What is the context of this PR?
Number is jumpy as it is updating server for every change. This changes so we now only sends the update on blur

Closes #483 

### How to review 
1. See that typing into routing number value is laggy when typing quickly
1. See that on this branch this is no longer true
